docs/content/components/center.mdx

```diff
@@ -0,0 +1,115 @@
+---
+title: Center
+---
+
+# Center
+
+## Description
+
+Center is a layout component that centers its children within itself.
+
+## Demo
+
+<MarigoldTheme>
+  <Center maxHeight="large" maxWidth="huge">
+    <Box border="1px solid #ced4da" bg="#e9ecef" height="100px" />
+  </Center>
+</MarigoldTheme>
+
+## Properties (API)
+
+| Property         | Type                           | Default  |
+| :--------------- | :----------------------------- | :------- |
+| `maxHeight`      | `string`                       |          |
+| `maxWidth`       | `string`                       |          |
+| `textAlign`      | `left, center, right`          | `center` |
+| `space`          | `ResponsiveStyleValue<string>` | `none`   |
+| `superCenntered` | `boolean`                      | `false`  |
```

Comment:
Shouldn't the layout always be centered? "Max height" didn't make sense otherwise 🤔